MetaMask/metamask-mobile
# feat: Support token detection on linea

## **Description**

Enables token autodetection on Linea

## **Related issues**

## **Manual testing steps**

1. Restore a wallet containing tokens (like WETH, USDC) on Linea 
2. Tokens should be automatically detected and suggested for import (`N new tokens found in this account`) 
3. Click `Import tokens` to manually import
4. There should be a search box that suggests tokens as you type their symbols

## **Screenshots/Recordings**

### **Before**

https://github.com/MetaMask/metamask-mobile/assets/3500406/c3fb7c64-bf5b-4415-be31-a6bdb57c800a

### **After**

https://github.com/MetaMask/metamask-mobile/assets/3500406/4b1f5c12-4f41-4a8a-891d-04b0fb6dd03d

## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
